docs/runtime_suite/mailchimp-notification-service/10_overview.md

```diff
@@ -0,0 +1,188 @@
+
+# 1) Sending transactional emails
+
+Transactional email service is provided with 2 main endpoints:
+
+- **Synchronous**: waits for Mailchimp/Mandrill to have actually queued the transactional email request
+
+- **Asynchronous**: fire and forget. The endpoint immediately returns, and the request is sent to Mailchimp/Mandrill but the service will not wait for Mailchimp/Mandrill response
+
+## Synchronous transactional email send
+
+The service is provided through the `POST /transactional/send-template` endpoint.
+
+An example JSON body is presented below:
+
+```
+{
+	"templateName": "my_template",
+	"subject": "Hello world!",
+	"fromEmail": "example@test.com",
+	"fromName": "Visible subject name",
+	"templateMergeVars": [
+		{
+            "name": "HELLO_WORLD",
+            "content": "I can provide the content I want here"
+		},
+	 ],
+     "to": [
+           {
+               "email": "receiver@test.com",
+               "type": "to"
+           },
+           {
+               "email": "cc@test.com",
+               "type": "cc"
+           },
+           {
+               "email": "bcc@test.com",
+               "type": "bcc"
+           }
+     ],
+     "attachments": [
+        {
+           "mimeType": "application/pdf",
+           "visibleName": "example.pdf",
+           "base64Content": "base64encodedContent",
+        }
+     ]
+}
+```
+
+- `templateName`: name of the email template as created in Mailchimp `Content -> Email Templates` section.
+  - NB: a `Send To Mandrill` action on the template from the Mailchimp UI is required to be able to send transactional emails using the template defined
+- `subject`: Email subject
+- `fromEmail`: this is the email sender address
+- `fromName`: this is the email friendly sender name
+- `templateMergeVars`: array body parameter
+  - each array entry defines an email `MERGE TAG`: in case the transactional email requires custom parameters inside its body, those can be provided using the `templateMergeVars` array
+  - each entry contains:
+    - `name`: name of the merge tag on Mailchimp email template. Here an example email template on Mailchimp, with an `HELLO_WORLD` merge tag:
+    ```
+    My beatiful email has a merge tag *|HELLO_WORLD|*.
+    Merge tags inside the Mailchimp email template must be within *| and |*.
+    HELLO_WORLD can be filled with the string content I need.
+    ```
+    - `content`: string content to be placed inside the merge tag specified by the parameter `name`
+- `to`: array body parameter
+  - multiple `to` array items can be specified
+  - `type`: can be one among `to|cc|bcc`
+  - `email`: email for which the `type` parameter applies
+- `attachments`:
+  - is an array to push attachments to the email to be sent
+  - every array item must have the following fields:
+    - `mimeType`: attachment MIME Type
+    - `visibleName`: attachment name which will be visible in the email template
+    - `base64Content`: attachment Base 64 content
```

Comment:
```suggestion
  - each array entry defines an email `MERGE TAG`: in case the transactional email requires custom parameters inside its body, those can be provided using the `templateMergeVars` array
  - each entry contains:
    - `name`: name of the merge tag on the Mailchimp email template. Here is an example email template on Mailchimp, with a `HELLO_WORLD` merge tag:
    ```
    My beautiful email has a merge tag *|HELLO_WORLD|*.
    Merge tags inside the Mailchimp email template must be within *| and |*.
    HELLO_WORLD can be filled with the string content I need.
    ```
    - `content`: string content to be placed inside the merge tag specified by the parameter `name`
- `to`: array body parameter
  - multiple `to` array items can be specified
  - `type`: can be one among `to|cc|bcc`
  - `email`: email for which the `type` parameter applies
- `attachments`:
  - is an array to push attachments to the email to be sent
  - every array item must have the following fields:
    - `mimeType`: attachment MIME Type
    - `visibleName`: attachment name which will be visible in the email template
    - `base64Content`: attachment Base 64 content
```